sympy/solvers/diophantine/diophantine.py

```diff
@@ -1310,6 +1310,10 @@ def diophantine(eq, param=symbols("t", integer=True), syms=None,
     {(0, n1, n2), (t_0, t_1, 2*t_0 + 3*t_1)}
     >>> diophantine(x**2 + 3*x*y + 4*x)
     {(0, n1), (-3*t_0 - 4, t_0)}
+    >>> diophantine(4*x + 6*y - 8)
+    {(3*t_0 - 4, 4 - 2*t_0)}
+    >>> diophantine(x**2 + y**2 - 13)
+    {(-3, -2), (-3, 2), (-2, -3), (-2, 3), (2, -3), (2, 3), (3, -2), (3, 2)}
```

Comment:
It is not clear to me why this needs to be shown in addition to the existing examples.